Pocket.Logger/Tests/ConfirmationLoggerTests.cs

```diff
@@ -186,10 +186,10 @@ public void After_Succeed_is_called_then_a_ConfirmationLogger_does_not_log_on_di
             using (var operation = Log.ConfirmOnExit())
             {
                 operation.Succeed("success!");
-                await Task.Delay(100);
+                await Task.Delay(150);
```

Comment:
`Task.Delay(n)` doesn't always sleep the thread for `n` milliseconds (at least not in a way that is recognizable from querying the clock).  

This change overshoots it by quite a bit.